js_modules/dagit/packages/core/src/nav/useRepositoryLocationReload.tsx

```diff
@@ -208,11 +208,12 @@ export const useRepositoryLocationReload = ({
   const {mutating, pollStartTime, error, errorLocationId} = state;
   const reloading = mutating || pollStartTime !== null;
 
-  return React.useMemo(() => ({reloading, error, errorLocationId, tryReload}), [
+  return React.useMemo(() => ({reloading, error, errorLocationId, tryReload, mutating}), [
```

Comment:
Is the `reloading` value insufficient?

Reply:
basically i tried that and didn't get the behavior i expeced, it seems like `pollStartTime` doesnt clear when I would expect it to. I think there are corner cases where it may be valid to submit another redeploy request while we are in the polling state so i didnt chase it down further and opted to just expose when the mutation request is inflight.